AsyncDisplayKit/ASCellNode.m

```diff
@@ -134,6 +141,13 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event
   [(_ASDisplayView *)self.view __forwardTouchesCancelled:touches withEvent:event];
 }
 
```

Comment:
Somewhere this file, we need to implement a set method for the layout delegate, so that when a layout delegate is provided, we can check if it responds to the scroll selector.

I am wondering though, what use case do you have for the layout delegate getting this call? Maybe we can avoid this completely for now, and require that developers implement a subclass of the cell to listen to the method, and forward it to a delegate of its own creation and if they will